config/crd/bases/configuration.konghq.com_kongclusterplugins.yaml

```diff
@@ -11,7 +16,6 @@ spec:
     shortNames:
     - kcp
     singular: kongclusterplugin
-  preserveUnknownFields: false
```

Comment:
If my memory serves me, we very specifically _had_ to add this in order to make migrating from older CRDs work correctly.